apps/governance/src/routes/home/index.tsx

```diff
@@ -163,11 +205,38 @@ const GovernanceHome = ({ name }: RouteChildProps) => {
       proposalsData
         ? getNotRejectedProposals<ProposalFieldsFragment>(
             proposalsData.proposalsConnection
-          ).slice(0, 3)
+          )
         : [],
     [proposalsData]
   );
 
+  const protocolUpgradeProposals = useMemo(
+    () =>
+      protocolUpgradesData
+        ? getNotRejectedProtocolUpgradeProposals<ProtocolUpgradeProposalFieldsFragment>(
```

Comment:
suggestion (non-blocking): i'd consider either using a different function here, where you don't need to double-filter it, or adding filter criteria (as a callback maybe?) to the function, so it can support your use-case here, where you want non-rejected proposals within a block height range.